h/static/images/icons/hypothesis.svg

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
If we're gonna do this as components then this image file should probably be in one folder along with the CSS and Jinja2/HTML for this "logged-out navbar" component.